sky/cli.py

```diff
@@ -725,34 +528,8 @@ def _parse_override_params(
     return override_params
 
 
-def _default_interactive_node_name(node_type: str):
-    """Returns a deterministic name to refer to the same node."""
-    # FIXME: this technically can collide in Azure/GCP with another
-    # same-username user.  E.g., sky-gpunode-ubuntu.  Not a problem on AWS
-    # which is the current cloud for interactive nodes.
-    assert node_type in _INTERACTIVE_NODE_TYPES, node_type
-    return f'sky-{node_type}-{common_utils.get_cleaned_username()}'
-
-
-def _infer_interactive_node_type(resources: sky.Resources):
-    """Determine interactive node type from resources."""
-    accelerators = resources.accelerators
-    cloud = resources.cloud
-    if accelerators:
-        # We only support homogenous accelerators for now.
-        assert len(accelerators) == 1, resources
-        acc, _ = list(accelerators.items())[0]
-        is_gcp = cloud is not None and cloud.is_same_cloud(sky.GCP())
-        if is_gcp and 'tpu' in acc:
-            return 'tpunode'
-        return 'gpunode'
-    return 'cpunode'
-
-
-def _check_resources_match(backend: backends.Backend,
-                           cluster_name: str,
-                           task: 'sky.Task',
-                           node_type: Optional[str] = None) -> None:
+def _check_resources_match(backend: backends.Backend, cluster_name: str,
+                           task: 'sky.Task') -> None:
     """Check matching resources when reusing an existing cluster.
 
     The only exception is when [cpu|tpu|gpu]node -c cluster_name is used with no
```

Comment:
Should update?

Reply:
Actually, this feels like a shallow func now, can inline.

Reply:
Good catch! Made it inline